src/OrchardCore.Modules/OrchardCore.AdminDashboard/Controllers/DashboardController.cs

```diff
@@ -85,6 +85,7 @@ public async Task<IActionResult> Index()
             return View(model);
         }
 
+        [Admin("dashboard/manage")]
```

Comment:
I am wondering if we should also specify the old route name in case some are using it in their app for routing. Not just here, but maybe everywhere

For example,
```
[Admin("dashboard/manage", "AdminDashboard")]
```

Reply:
This will be a lot of (non-trivial) work, especially after the fact. So before I get into it, can you think of an example how that name may be used? I can't think of one.

Reply:
yes. look at my next comment. 
When you specify this

```
routes.MapAreaControllerRoute(
    name: "AUniqueNameToTheRoute", // This is the route name. This value is something we can use to route the user to
    // ....
);
```

You can then use that value to redirect. For example `RedirectToRoute("AUniqueNameToTheRoute")`. If someone in their project is already doing route name routing, their code will break after merging this PR.